app/views/purchase/create.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1 Purchase#create
+%p Find me in app/views/purchase/create.html.haml
```

Comment:
Do you need this file?
If you do give it some other content

Reply:
@ESteinman ?? Wanna look into this?